src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -1234,6 +1253,21 @@
       "dataType": "bool",
       "value": "(httpEvaluator && useConfiguration)"
     },
+    "useHttpDefault": {
+      "type": "computed",
+      "datatype": "bool",
+      "value": "useHttp && http == 'default'"
+    },
+    "useHttpRefit": {
```

Comment:
For now, this one should be used as what gets selected by default for the recommended template since the recommended template code is using Refit out of the box